### DISCUSSION TITLE
customized sound once tap your card

### DISCUSSION THREAD

**Discussion by winsanmwtv** [Limaru-Minecraft/iciwi#63] [Ideas]
as it said

**Comment by Mineshafter61** [comment]
AAAAAAAAAA -oh wait i think this may be doable

**Comment by Mineshafter61** [comment]
There's a few ways:
- a single global sound (easiest)
- per company
- per sign
- make it extremely complicated for both me and server admins

**Comment by Mineshafter61** [comment]
done
